workflow/tests/base_results/results_simulations_misc.csv

```diff
@@ -1,508 +1,508 @@
+base-appliances-coal.xml,0.0,0.0,0.0,1354.7,998.0,11171.6,2563.5,1984.0,3545.9,3545.9,1984.0,3545.9,3545.9,23.636,19.047,0.0
+base-appliances-dehumidifier-ief-portable.xml,0.0,0.0,0.0,1354.7,998.0,9789.3,2412.1,2117.1,2959.0,2959.0,2117.1,2959.0,2959.0,9.973,15.465,0.0
+base-appliances-dehumidifier-ief-whole-home.xml,0.0,0.0,0.0,1354.7,998.0,9789.3,2412.1,2066.1,2959.0,2959.0,2066.1,2959.0,2959.0,9.974,15.465,0.0
+base-appliances-dehumidifier-multiple.xml,0.0,0.0,0.0,1354.7,998.0,9789.3,2412.1,1947.2,2959.0,2959.0,1947.2,2959.0,2959.0,9.997,15.465,0.0
+base-appliances-dehumidifier.xml,0.0,0.0,0.0,1354.7,998.0,9789.3,2412.1,1930.7,2959.0,2959.0,1930.7,2959.0,2959.0,9.99,15.465,0.0
+base-appliances-freezer-temperature-dependent-schedule.xml,0.0,0.0,0.0,1354.7,998.0,11171.6,2563.5,2131.1,3729.0,3729.0,2131.1,3729.0,3729.0,23.553,19.012,0.0
+base-appliances-gas.xml,0.0,0.0,0.0,1354.7,998.0,11171.6,2563.5,1984.0,3545.9,3545.9,1984.0,3545.9,3545.9,23.636,19.047,0.0
+base-appliances-modified.xml,0.0,0.0,0.0,1354.7,1998.3,11171.6,2563.5,2109.7,3863.9,3863.9,2109.7,3863.9,3863.9,23.62,18.864,0.0
+base-appliances-none.xml,0.0,0.0,0.0,0.0,0.0,11171.5,2563.5,1817.1,3042.4,3042.4,1817.1,3042.4,3042.4,24.034,17.656,0.0
+base-appliances-oil.xml,0.0,0.0,0.0,1354.7,998.0,11171.6,2563.5,1984.0,3545.9,3545.9,1984.0,3545.9,3545.9,23.636,19.047,0.0
+base-appliances-propane.xml,0.0,0.0,0.0,1354.7,998.0,11171.6,2563.5,1984.0,3545.9,3545.9,1984.0,3545.9,3545.9,23.636,19.047,0.0
+base-appliances-refrigerator-temperature-dependent-schedule.xml,0.0,0.0,0.0,1354.7,998.0,11171.6,2563.5,2082.5,3612.6,3612.6,2082.5,3612.6,3612.6,23.71,18.744,0.0
+base-appliances-wood.xml,0.0,0.0,0.0,1354.7,998.0,11171.6,2563.5,1984.0,3545.9,3545.9,1984.0,3545.9,3545.9,23.636,19.047,0.0
+base-atticroof-cathedral.xml,0.0,0.0,0.0,1354.7,998.0,11171.5,2563.5,2087.2,3299.9,3299.9,2087.2,3299.9,3299.9,23.039,16.736,0.0
+base-atticroof-conditioned.xml,0.0,0.0,0.0,1354.7,998.0,11171.6,2471.3,2327.2,4208.8,4208.8,2327.2,4208.8,4208.8,23.048,20.694,0.0
+base-atticroof-flat.xml,0.0,0.0,0.0,1354.7,998.0,11171.5,2563.5,2067.9,3142.7,3142.7,2067.9,3142.7,3142.7,17.971,13.003,0.0
+base-atticroof-radiant-barrier-ceiling.xml,0.0,0.0,0.0,1354.7,998.0,9789.3,2412.1,1756.0,3475.9,3475.9,1756.0,3475.9,3475.9,14.605,20.04,0.0
+base-atticroof-radiant-barrier.xml,0.0,0.0,0.0,1354.7,998.0,9789.2,2412.1,1747.6,3433.3,3433.3,1747.6,3433.3,3433.3,13.691,19.088,0.0
+base-atticroof-unvented-insulated-roof.xml,0.0,0.0,0.0,1354.7,998.0,11171.6,2563.5,2069.5,3413.5,3413.5,2069.5,3413.5,3413.5,20.537,14.701,0.0
+base-atticroof-vented.xml,0.0,0.0,0.0,1354.7,998.0,11171.5,2563.5,2083.8,3399.5,3399.5,2083.8,3399.5,3399.5,23.824,16.752,0.0
+base-battery-scheduled-power-outage.xml,0.0,5.0,0.0,1241.4,914.9,10291.7,2361.6,7802.3,8148.5,8148.5,7802.3,8148.5,8148.5,23.712,21.153,1.337
+base-battery-scheduled.xml,0.0,0.0,0.0,1354.7,998.0,11171.6,2563.5,7802.1,8197.2,8197.2,7802.1,8197.2,8197.2,23.71,18.744,1.438
```

Comment:
Battery charging/discharging is being accounted for again. (Was removed in the EV PR.)

Reply:
@joseph-robertson The reason the increase is ~4.5 kW instead of the battery's rated power of ~6 kW is because the scheduled charging is not coincident with the building peak. (In the chart below, orange is battery charging/discharging, blue is building consumption, and the results are stacked.)

![image](https://github.com/user-attachments/assets/0d96c55c-1c84-42f0-a845-ce9e148e69d3)

Reply:
Makes sense. Only if the scheduled charging was coincident with building peak would the increases shown in `results_simulation_misc.csv` be equal to the battery's rated power. (Does this mean rated power input/charge is assumed to be equal to rated power output/discharge? Looking at example IDF and [this](https://github.com/NREL/OpenStudio-HPXML/blob/master/HPXMLtoOpenStudio/resources/battery.rb#L86), it appears this is the case -- should the docs note anything about this?)